2.4/examples/replica/mongodb-clustered.json

```diff
@@ -163,6 +168,10 @@
                   {
                     "name": "MONGODB_KEYFILE_VALUE",
                     "value": "${MONGODB_KEYFILE_VALUE}"
+                  },
+                  {
```

Comment:
This part is missing in #223. @git001 Please add it (TIMEZONE has no effect without this).